src/components/MainPage.jsx

```diff
@@ -94,10 +96,10 @@ function MainPage(props) {
   ];
 
 
-  const onLogoClick = useCallback(() => {
-    onSelectedTabChanged("event");
-    // mixpanel.track("Rainbow logo clicked") // Tracking logo click event
-  }, []);
+  //const onLogoClick = useCallback(() => {
```

Comment:
@MaximSamorukov can you please comment this back out? We want to leave this in as an example in case someone would want to implement that code, but not actually make it active. Feel free to add a comment above for clarity

Reply:
@brittanyjoiner15 should I leave onSelectedTabChange commented? It is not defined and eslint emits an error.

![image](https://user-images.githubusercontent.com/53444059/206987686-692c9ddc-7e9a-4ca8-b343-635c1e3589a0.png)